app/serializers/question_serializer.rb

```diff
@@ -1,8 +1,28 @@
 class QuestionSerializer < ActiveModel::Serializer
   attributes :id, :user_id, :title, :content, :votes, :comments, :tags, :answers,
-             :created_at, :updated_at
+             :created_at, :updated_at, :votes_count, :answers_count, :views_count,
+             :time_updated, :user
 
-   has_many :comments, as: :comment_on
-   has_many :tags, as: :subscriber
-   has_many :answers
+  has_many :comments, as: :comment_on
+  has_many :tags, as: :subscriber
+  has_many :answers
+
+  def votes_count
+    object.votes.count
+  end
+
+  def answers_count
+    object.answers.count
+  end
+
+  def views_count
```

Comment:
Just thinking; is there any need for creating this as a separate method in the model? Given that it simply circles back to call _just_ one column in same model table :thought_balloon: